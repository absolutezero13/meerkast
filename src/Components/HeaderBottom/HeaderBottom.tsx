import React from "react";
import "./HeaderBottom.css";

const HeaderBottom: React.FC = (props) => {
  console.log(props);
  return (
    <div className="header-bottom">
      <h3>Popular in Turkey</h3>
    </div>
  );
};

export default HeaderBottom;
