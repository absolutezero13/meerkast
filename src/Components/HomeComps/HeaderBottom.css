.header-bottom {
  display: flex;
  background: rgba(65, 65, 65, 255);
  color: white;
  height: 8vh;
  padding: 0 10%;
  align-items: center;
  box-shadow: 3px 3px 9px gray;
}
