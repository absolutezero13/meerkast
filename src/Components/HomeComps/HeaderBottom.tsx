import React from "react";
import "./HeaderBottom.css";
import { BrowserRouter, Switch, Route } from "react-router-dom";
const HeaderBottom: React.FC = () => {
  return (
    <BrowserRouter>
      <div className="header-bottom">
        <h2>Popular in Turkey </h2>
      </div>
    </BrowserRouter>
  );
};

export default HeaderBottom;
