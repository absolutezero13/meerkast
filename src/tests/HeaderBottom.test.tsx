import React from "react";
import { render } from "@testing-library/react";
import HeaderBottom from "../Components/HeaderBottom/HeaderBottom";

const { container } = render(<HeaderBottom />);

// SnapShot Test
it("header bottom matches snapshot", () => {
  expect(container).toMatchSnapshot();
});
