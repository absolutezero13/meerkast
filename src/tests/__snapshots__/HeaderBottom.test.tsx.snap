// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header bottom matches snapshot 1`] = `
<div>
  <div
    class="header-bottom"
  >
    <h3>
      Popular in Turkey
    </h3>
  </div>
</div>
`;
